Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python:3-alpine
+
+RUN mkdir -p /src/scrubadubdub && mkdir -p /src/test
+
+COPY scrubadubdub /src/scrubadubdub
+COPY tests /src/tests
+WORKDIR /src
+
+COPY setup.py setup.py
+COPY README.md README.md
+
+RUN pip install --upgrade setuptools wheel cloudsmith-cli
+RUN python ./setup.py sdist bdist_wheel
+RUN python -m unittest discover -s tests
+
+CMD cloudsmith push python carta/pip dist/*.whl
```

I think it's cool! I might change this line to `--help` and put the actual package publishing command in the README?

```suggestion
CMD cloudsmith --help
```

---

Sounds good, I'll change that up and amend the readme with the steps